submissions/sofiichuk/pop-up-task/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <title>dropdown</title>
+    <meta charset="UTF-8">
+</head>
+<body>
+<nav>
+    <ul class="navBar">
+        <li class="navBar__item item_txt"><a href="#">Mail</a></li>
+        <li class="navBar__item item_txt"><a href="#">Images</a></li>
+        <li class="navBar__item item_btn">
+            <input id="input_1" type="checkbox">
+            <label for="input_1" id="label_1">
+                <img alt="dropdown menu" src="icons/dropdown_menu.png">
+            </label>
+                <ul class="dropdownMenu">
```

Comment:
Houston we have a problem
![image](https://user-images.githubusercontent.com/16196199/188207110-5caa263d-01b3-4cd7-9875-aa3c7ae050e8.png)

![image](https://user-images.githubusercontent.com/16196199/188207173-d1951f5a-f40c-46aa-9704-1dc3c15b7df1.png)

Reply:
Fixed that few minutes ago)